### DISCUSSION TITLE
Contributing gatsby-source-hashnode

### DISCUSSION THREAD

**Discussion by nitzano** [gatsbyjs/gatsby#30022] [Community]
Hi everyone, 

I'm the author of [gatsby-source-hashnode](https://www.npmjs.com/package/gatsby-source-hashnode), a plugin that connects Gatsby and [Hashnode](https://hashnode.com) together. 

Full source code can be found here: https://github.com/nitzano/gatsby-source-hashnode

Contains features as: 

* Getting all the user's hashnode posts.
* Calculating the estimated reading time.
* Getting user details.

I would more than happy to make this an official gatsby plugin and would like to know
what's the next steps for this to happen. 

Some more details:

* MIT licened
* The plugin already contains tests and and dependecies as other official plugins.
* An article how it was created: [Adding hashnode to Gatsby websites](https://blog.nitzano.com/adding-hashnode-to-your-gatsby).

I'd be happy to open PR with the plugin (any help will be welcomed).

**Comment by LekoArts** [comment]
Hi!

Thanks for creating the plugin :) We don't add any plugins to our monorepo anymore. As long as you have the `gatsby-plugin` keyword (which you have: https://github.com/nitzano/gatsby-source-hashnode/blob/48cdd65b6fe590d40f10818c912c55ed5b708e97/packages/gatsby-source-hashnode/package.json#L16) it'll show up in the search.

**Reply by nitzano** [reply]
Thanks @LekoArts. I'll keep updating via npm
